magento/pwa-studio
# [PWA-983] Migrate gql related files and folders to peregrine

## Description

This PR moves all `.gql` files that are in CartPage components to their corresponding talons in `@magento/peregrine` and update respective talons.

## Related Issue

Closes PWA-983

## Acceptance

### Verification Stakeholders

### Specification

## Verification Steps

- Cart page `.gql.js` files are moved to Peregrine and no longer reside in Venia
- The Cart page functions as expected
- Signing in on the Cart page merges carts correctly

#### Test scenario(s) for direct fix/feature

- Verify that you can edit cart items successfully as a guest user
- Verify that you can edit cart items successfully as an authorized user
- Verify that you can come to the cart page as a guest user, sign in, and that your carts are merged correctly and you can check out successfully

#### Test scenario(s) for any existing impacted features/areas

#### Test scenario(s) for any Magento Backend Supported Configurations

#### Is Browser/Device testing needed?

#### Any ad-hoc/edge case scenarios that need to be considered?

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
